src/lib/server.ts

```diff
@@ -408,27 +445,41 @@ export class ZwavejsServer extends EventEmitter {
   private defaultHost: string = "0.0.0.0";
   private responder?: Responder;
   private service?: CiaoService;
+  private remoteController?: ZwavejsRemoteController;
```

Comment:
It's initialized in the constructor
```suggestion
  private remoteController: ZwavejsRemoteController;
```